### PR TITLE
Fix isCommand check and set min rows threshold for saveToFile

### PR DESCRIPTION
**Pull request by turboFei**
# :mag: Description
## Issue References 🔗

This pull request fixes #
I found that, with saveToFile enabled with the default min size threshold, even I run a simple `set` command, It also save the result to file.
image

I think we need to skip this kind of queries.

## Describe Your Solution 🔧

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Types of changes :bookmark:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Related Unit Tests

---

# Checklist 📝

- [ ] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

**Comment by turboFei** [comment]
cc @cxzl25 @wForget @pan3793

**Comment by turboFei** [comment]
thanks @wForget , updated

**Review by wForget — APPROVED** [review]
LGTM

**Review by cxzl25 — APPROVED** [review]

File: externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala [hunk 1]
```diff
@@ -270,7 +281,8 @@ object SparkDatasetHelper extends Logging {
     minSize > 0 && colSize > 0 && stats >= minSize
   }
 
-  private def isCommandExec(nodeName: String): Boolean = {
+  def isCommandExec(result: DataFrame): Boolean = {
+    val nodeName = result.queryExecution.executedPlan.getClass.getName
```
[the remark this answers is not in the document]
Comment (turboFei) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala:285]: spark 3.1

scala> spark.sql("set").queryExecution.executedPlan.nodeName
res22: String = Execute SetCommand

scala> spark.sql("show tables").queryExecution.executedPlan.nodeName
res23: String = Execute ShowTablesCommand

scala>

spark 3.5

image

[the remark this answers is not in the document]
Comment (turboFei) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala:285]: it is a bug and not covered by UT before.

[the remark this answers is not in the document]
Comment (turboFei) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala:285]: it doesn't work for spark 3.5 as well

**Pull request closed without merging**